lib/data/external/AwsClient.js

```diff
@@ -127,11 +130,11 @@ class AwsClient {
             VersionId: dataStoreVersionId,
             Range: range ? `bytes=${range[0]}-${range[1]}` : null,
         }).on('success', response => {
-            log.trace('AWS GET request response headers',
+            log.trace(`${this._type} GET request response headers`,
```

Comment:
We don't string concat in any error messages for perf reasons. Please put the variable in the object argument -- backendType: this._type.  Please change throughout file for any other logger usages.

Reply:
I see. I will take not of that and make the proper changes.